lightcrafts/src/main/java/com/lightcrafts/model/ImageEditor/ZoneFinder.java

```diff
@@ -18,6 +18,7 @@
 import javax.media.jai.LookupTableJAI;
 import javax.media.jai.PlanarImage;
 import javax.media.jai.RenderedOp;
+import javax.swing.*;
```

Comment:
Use single class import.

```suggestion
import javax.swing.SwingUtilities;
import javax.swing.Timer;
```